specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-05-01/managedClusters.json

```diff
@@ -3681,21 +3627,22 @@
         "name",
         "namespace",
         "identity"
-      ]
+      ],
+      "description": "Details about the pod identity assigned to the Managed Cluster."
     },
     "ManagedClusterPodIdentityException": {
```

Comment:
I don't quite understand why you would want one of these. Need to talk to something to understand and then write a doc accordingly.